NGitLab/Impl/Json/Int64Converter.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace NGitLab.Impl.Json
+{
+    internal sealed class Int64Converter : JsonConverter<long>
```

Comment:
Could you add a test to validate this converter is not used for `long?`?

Reply:
According to the [System.Text.Json documentation](https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-converters-how-to?pivots=dotnet-6-0#handle-null-values), this custom converter will **not** be used for `Nullable<long>`:

```
By default, the serializer handles null values as follows:
- For reference types and Nullable<T> types:
  - It does not pass null to custom converters on serialization.
  - It does not pass JsonTokenType.Null to custom converters on deserialization.
  - It returns a null instance on deserialization.
  - It writes null directly with the writer on serialization.
- For non-nullable value types:
  - It passes JsonTokenType.Null to custom converters on deserialization. (If no custom converter is available, a JsonException exception is thrown by the internal converter for the type.)
```

So unless you think the documentation is lying 😁, I don't see the need for such a test.

Reply:
Let's trust the documentation!